benchmark/src/bin/ruben.rs

```diff
@@ -132,27 +123,28 @@ fn main() {
     info!("RuBen: the utility to (Ru)n (Ben)chmarker");
     let args = Opt::new_from_args();
     info!("Parsed arguments: {:#?}", args);
-
     try_start_metrics_server(&args);
     let mut bm = create_benchmarker_from_opt(&args);
-    let mut wallet = WalletLibrary::new();
-    let mut accounts = gen_accounts(&mut wallet, args.num_accounts);
-    bm.mint_accounts(&args.faucet_key_file_path, &accounts);
-    match args.executable {
-        Executable::TestLiveness => {
-            test_liveness(
+    let mut faucet_account = bm.load_faucet_account(&args.faucet_key_file_path);
+    match args.txn_pattern {
```

Comment:
you can do something like
```
let generator = match args.txn_pattern {
    TransactionPattern::Ring => RingTransferTxnGenerator::new(),
    TransactionPattern::Pairwise => PairwiseTransferTxnGenerator::new(),
};
measure_throughput(bm, generator, ...);
```

Reply:
This is exactly my first version and it failed to pass compiler because the `match` can only return 1 type, and it doesn't make sense to create an enum to unify different generators as we already have a Trait. So now I have a solution: using `Box<dyn TraitName>`.